patch-place-break/patch-place-break-config/src/test/java/fr/djaytan/minecraft/jobsreborn/patchplacebreak/config/annotated/DbmsHostValidatingPropertiesTest.java

```diff
@@ -25,7 +25,9 @@
 package fr.djaytan.minecraft.jobsreborn.patchplacebreak.config.annotated;
```

Comment:
A merge between DbmsHostValidatingPropertiesTest and DbmsHostValidatingPropertiesIT